circle.yml

```diff
@@ -10,6 +10,7 @@ machine:
   environment:
     DOCKER_GENERATOR_IMAGE_NAME: swaggerapi/swagger-generator
     DOCKER_CODEGEN_CLI_IMAGE_NAME: swaggerapi/swagger-codegen-cli
+    DOCKER_BUILD_TAG: 2.2.3
```

Comment:
@jebentier thanks for the enhancement. Should this be 2.3.0-SNAPSHOT instead? (the latest master is 2.3.0)

Reply:
Typically the image tags should match the git tags. To make it dynamic using a git command like such:

```
tag=$(git tag -l --points-at HEAD) ; if [[ -z "$tag" ]]; then echo "latest" ; else echo "$tag" ; fi
```

If there is no tag on the current commit, you get back "latest" otherwise you get the tag which would have been `v2.2.3` if that commit was the one being built.

Reply:
This could be done more easily in Docker Hub as an automated build. Probably a silly question, but @wing328 do you know if there's a reason it's set up in CI rather than in Docker Hub?

Reply:
@jimschubert sorry I don't have an answer as it's setup by Tony.

cc @fehguy

Reply:
@kenjones-cisco thanks for the explanation. If it's within Docker user's expectation that 2.2.3 can refer to the Docker image built by the latest master (2.3.0), then it's totally fine with me.